File: Shared/Dto/C2D2R/DetachOtauDto.cs
using System;
using System.Runtime.Serialization;

namespace Iit.Fibertest.Dto
{
    [DataContract]
    public class DetachOtauDto
    {
        [DataMember]
        public string ClientIp { get; set; }

        [DataMember]
        public Guid RtuId { get; set; }

        [DataMember]
        public Guid OtauId { get; set; }

        [DataMember]
        public NetAddress OtauAddress { get; set; }

        [DataMember]
        public int OpticalPort { get; set; }
    }
}

File: Client/WebClient/src/app/models/dtos/rtu/detachOtauDto.ts
import { RtuMaker } from "../../enums/rtuMaker";
import { NetAddress } from "../../underlying/netAddress";

export class DetachOtauDto {
  connectionId: string;
  otauId: string;
  rtuId: string;
  rtuMaker: RtuMaker;
  netAddress: NetAddress;
  opticalPort: number;
}
